src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Extensions/ParenthesizedExpressionSyntaxExtensions.cs

```diff
@@ -40,6 +40,12 @@ internal static class CommonParenthesizedExpressionSyntaxExtensions
                 }
             }
 
+            // See https://github.com/dotnet/roslyn/issues/47365
```

Comment:
i would prefer an actual comment here.  having a link is fine as well.

Reply:
@CyrusNajmabadi Like this?

```suggestion
            // It's not safe to change associativity for dynamic variables as the actual type isn't known. See https://github.com/dotnet/roslyn/issues/47365
```